src/core_plugins/kbn_vislib_vis_types/public/controls/point_series/series.html

```diff
@@ -24,8 +26,8 @@
           ng-class="{ 'fa-caret-down': isSeriesOpen, 'fa-caret-right': !isSeriesOpen }"
           class="kuiIcon fa-caret-right kuiSideBarCollapsibleTitle__caret"
         ></span>
-        <span class="kuiSideBarCollapsibleTitle__text">
-          {{chart.data.label}}
+        <span class="kuiSideBarCollapsibleTitle__text">	        
```

Comment:
This looks like you added a couple of trailing whitspace, which should be removed.